packages/core/src/EventManager.ts

```diff
@@ -69,6 +69,22 @@ export class EventManager {
                     payload.key === 'u'
                 ) {
                     return this.editor.execCommand('applyFormat', { format: 'underline' });
+                } else if (
+                    payload.ctrlKey &&
+                    !payload.altKey &&
+                    payload.shiftKey &&
+                    !payload.metaKey &&
+                    payload.key === '7'
+                ) {
+                    return this.editor.execCommand('list', { type: 'ordered' });
+                } else if (
+                    payload.ctrlKey &&
+                    !payload.altKey &&
+                    payload.shiftKey &&
+                    !payload.metaKey &&
+                    payload.key === '8'
+                ) {
+                    return this.editor.execCommand('list', { type: 'unordered' });
```

Comment:
I would add `TODO: remove this when we have a toolbar.` Because these are not "standardly recognized" shortcuts.